lib/statsd/class-statsd.php

```diff
@@ -160,5 +164,7 @@ public function send( $data, $sample_rate = 1 ) {
 			$escaped_url = addslashes( $url );
 			trigger_error( "Statsd::send exception('$escaped_url'): {$e->getMessage()}", \E_USER_WARNING ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		}
+
+	// phpcs:enable
```

Comment:
Can you please point me towards where the earlier comment of PHPCS was disabled that it needs to be re-enabled again?

Reply:
[Line 99](https://github.com/Automattic/vip-go-mu-plugins/pull/2415/files/681a2b84aa40e4161fa2556719e39c845351d82c#diff-643cc2c463824e2bd9405cdd227d792ec8a63bb6430187d9d33b9a30c92a6b85R99)

Reply:
Thanks!